lightning/src/chain/keysinterface.rs

```diff
@@ -238,7 +255,8 @@ impl_writeable!(InMemoryChannelKeys, 0, {
 	payment_base_key,
 	delayed_payment_base_key,
 	htlc_base_key,
-	commitment_seed
+	commitment_seed,
+	remote_funding_pubkey
```

Comment:
do we need to worry about backwards compat since this is serialized?

Reply:
I would say no, we have been pretty liberal until now updating other struct like ChannelManager and ChannelMonitor though we may should start soon to version our serialization to be consistent